packages/fxa-auth-server/test/local/payments/fixtures/customer1.json

```diff
@@ -49,6 +49,7 @@
         "id": "sub_test1",
         "object": "subscription",
         "application_fee_percent": null,
+        "created": 1582765012,
```

Comment:
The test data excludes several fields that are included in the response, I updated the mock data to include the `created` field.